modules/ztp-precaching-troubleshooting.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-pre-staging-troubleshooting_{context}"]
+= Troubleshooting
+
+== Rendered catalog is invalid
+
+When you download images by using a local or disconnected registry, you might see the `The rendered catalog is invalid` error. This means that you are missing certificates of the new registry you want to pull content from.
+
+[NOTE]
+====
+The {factory-prestaging-tool} image is built on a UBI {op-system-base} image. Certificate paths and locations are the same on {op-system}.
+====
+
+.Example error
+[source,terminal]
+----
+Generating list of pre-cached artifacts...
+error: unable to run command oc-mirror -c /mnt/imageset.yaml file:///tmp/fp-cli-3218002584/mirror --ignore-history --dry-run: Creating directory: /tmp/fp-cli-3218002584/mirror/oc-mirror-workspace/src/publish
+Creating directory: /tmp/fp-cli-3218002584/mirror/oc-mirror-workspace/src/v2
+Creating directory: /tmp/fp-cli-3218002584/mirror/oc-mirror-workspace/src/charts
+Creating directory: /tmp/fp-cli-3218002584/mirror/oc-mirror-workspace/src/release-signatures
+backend is not configured in /mnt/imageset.yaml, using stateless mode
+backend is not configured in /mnt/imageset.yaml, using stateless mode
+No metadata detected, creating new workspace
+level=info msg=trying next host error=failed to do request: Head "https://eko4.cloud.lab.eng.bos.redhat.com:8443/v2/redhat/redhat-operator-index/manifests/v4.11": x509: certificate signed by unknown authority host=eko4.cloud.lab.eng.bos.redhat.com:8443
+
+The rendered catalog is invalid.
+
+Run "oc-mirror list operators --catalog CATALOG-NAME --package PACKAGE-NAME" for more information.
+
+error: error rendering new refs: render reference "eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/redhat-operator-index:v4.11": error resolving name : failed to do request: Head "https://eko4.cloud.lab.eng.bos.redhat.com:8443/v2/redhat/redhat-operator-index/manifests/v4.11": x509: certificate signed by unknown authority
+----
+
+. To solve this issue, first, copy the registry certificate into your server:
```

Comment:
Additionally, start your procedural steps with the verb: 

"Copy the registry cert..."